_pages/clients/r.md

```diff
@@ -56,207 +54,219 @@ If you are running [Algorithmia Enterprise](/enterprise), you can specify the AP
+{% highlight r %}
+algoJSON <- client$algo('nlp/LDA/1.0.0')
+input = list(
+    docsList = c(
+        "It is apple picking season",
+        "The apples are ready for picking"
+    )
+)
+response <- algoJSON$pipe(input)
+print(reponse$result)
+{% endhighlight %}
 
-Next check for the data collection's permission type and update those permissions to private:
+The output will be a list with named elements for each relevant word, and the number of occurrances:
 
 {% highlight r %}
-# Create the acl object and check if it's the ReadAcl.MY_ALGORITHMS default setting
-acl = nlp_directory$getPermissions()  # Acl object
-acl$read_acl # Returns "MY_ALGORITHMS" Acl Type
+[[1]]
+[[1]]$ready
+[1] 1
 
-# Update permissions to private
-nlp_directory$update_permissions(ReadAcl.PRIVATE)
-nlp_directory$get_permissions()$read_acl # Returns "PRIVATE" Acl Type
-{% endhighlight %}
 
-Notice that we changed our data collection to private, which means that only you will be able to read and write to your data collection.
+[[2]]
+[[2]]$apples
+[1] 1
 
-Note that read access that is set to the default `DataMyAlgorithms` allows any algorithm you call to have access to your data collection so most often, this is the setting you want when you are calling an algorithm and are an application developer.
 
-For more information on collection-based Access Control Lists (ACLs) and other data collection permissions go to the [Hosted Data Guide]({{site.baseurl}}/data/hosted).
+[[3]]
+[[3]]$season
+[1] 1
 
-### Upload Data to your Data Collection
 
-So far you've created your data collection and checked and updated directory permissions. Now you're ready to upload the text file that you created at the beginning of the guide to your data collection using the Data API.
+[[4]]
+[[4]]$apple
+[1] 1
+
+[[4]]$picking
+[1] 2
+{% endhighlight %}
+
+You might have noticed that in this example we included a version number when instantiating the algorithm. Pinning your code to a specific version of the algorithm can be especially important in a production environment where the underlying implementation might change from version to version.
 
-First create a variable that holds the path to your data collection and the text file you will be uploading:
+### Request Options
+
+The client exposes options that can configure algorithm requests. This includes support for changing the timeout or indicating that the API should include `stdout` in the response. In the following example, we set the timeout to 60 seconds and disabling `stdout` in the response:
 
 {% highlight r %}
-text_file <- "data://YOUR_USERNAME/nlp_directory/jack_london.txt"
+algo$setOptions(timeout=60, stdout=FALSE)
 {% endhighlight %}
 
-Next upload your local file to the data collection using the `.putFile()` method:
+You can find more details in [API Docs](/developers/api/?python) > [Invoke an Algorithm](/developers/api/?python#invoke-an-algorithm).
```

Comment:
Let's link to the curl example instead of python since R examples were removed a while back and haven't been replaced yet https://algorithmia.com/developers/api/?shell#invoke-an-algorithm